### PR TITLE
Localize an optimistic error that was only in English

### PR DESCRIPTION
**Pull request by yuwenmemon**
@nkuoch please review

HOLD ON COPY CONFIRMATION: https://github.com/Expensify/Expensify/issues/218743#issuecomment-1221652943

### Details
Fixed the local failure data error message to be localized instead of just in English.

### Fixed Issues
None, just a comment from @iwiznia here: https://github.com/Expensify/App/pull/10269#discussion_r949330820

### Tests
- Added in a throw in the `BeginSignIn` API command
- When I attempted to log in, made sure the error message showed correctly (see screenshot below)

### QA Steps
- None

### Screenshots
Screen Shot 2022-08-19 at 2 35 04 PM

### CODE REVIEW

File: src/languages/es.js [hunk 1]
```diff
@@ -485,6 +485,7 @@ export default {
         error: {
             invalidFormatEmailLogin: 'El email introducido no es válido. Corrígelo e inténtalo de nuevo.',
         },
+        cannotGetAccountDetails: 'No se pueden obtener los detalles de la cuenta. Por favor inténtalo de nuevo.',
```
Comment (iwiznia) [inline src/languages/es.js:488]: Why was this changed to plural? It sounds odd like this.

Comment (yuwenmemon) [reply src/languages/es.js:488]: That was just a placeholder I had from Google Translate

**Review by nkuoch — DISMISSED** [review]

**Comment by yuwenmemon** [comment]
@nkuoch updated spanish and removed HOLD

**Review by nkuoch — APPROVED** [review]
